dist/extension.js

```diff
@@ -0,0 +1,47 @@
+const vscode = require('vscode');
+
+const THEME_CONFIGURATION_ID = 'markdown-preview-github-styles';
+const THEME_CONFIGURATION_KEY = 'colorMode';
+
+const THEME_CONFIGURATION_VALUES = {
+    'auto': true,
+    'system': true,
+    'light': true,
+    'dark': true,
+    'high-contrast': true
+}
+const THEME_CONFIGURATION_AUTO = 'auto';
+
+let settings = vscode.workspace.getConfiguration(THEME_CONFIGURATION_ID, null);
```

Comment:
[nit] Instead of saving this as a global, just retrieve the setting using the VS Code api when needed. VS Code already caches the settings internally so the call should be fast

Reply:
Resolved in c4f0515